cernopendata/modules/theme/assets/semantic-ui/scss/search.scss

```diff
@@ -4,6 +4,27 @@
   }
 }
 
+.ui.card {
+  .content {
+    padding: 10px 15px !important;
```

Comment:
As mentioned [here](https://github.com/cernopendata/opendata.cern.ch/pull/3071/files#r577455237), it'd be nice to get rid of `!important`s.